mitosheet/src/components/taskpanes/Graph/GraphStyleTab.tsx

```diff
@@ -1,20 +1,264 @@
 // Copyright (c) Saga Inc.
 
 import React from 'react';
+import { GraphParams } from '../../../types';
+import Input from '../../elements/Input';
+import Toggle from '../../elements/Toggle';
+import Col from '../../spacing/Col';
 import Row from '../../spacing/Row';
 
 /* 
-    The tabs at the bottom of the column control panel that allows users to switch
-    from sort/filter to seeing summary statistics about the column
+    Contains all of the options for styling graphs,
+    like setting the title and axis labels
 */
-function GraphStyleTab(): JSX.Element {
+function GraphStyleTab(props: {
+    graphParams: GraphParams
+    setGraphParams: React.Dispatch<React.SetStateAction<GraphParams>>;
+    setGraphUpdatedNumber: React.Dispatch<React.SetStateAction<number>>;
+}): JSX.Element {
 
-    return (  
-        <Row justify='space-around'>
-            Coming soon!
-        </Row>
-        
+    const graphStylingParams = props.graphParams.graphStyling
+
+    return ( 
+        <div className='graph-sidebar-toolbar-content'>   
+            <div>
+                <div className='text-header-2'>
+                    Title
+                </div>
+                <Row justify='space-between' align='center'>
+                    <Col>
+                        <p>
+                            Title
+                        </p>
+                    </Col>
+                    <Input 
+                        value={graphStylingParams.title.title || ''}
+                        placeholder="Default Graph Title"
+                        onChange={(e) => {
+                            props.setGraphParams(prevGraphParams => {
+                                const graphParamsCopy: GraphParams = JSON.parse(JSON.stringify(prevGraphParams)); 
+                                const newTitle =  e.target.value !== '' ? e.target.value : undefined
```

Comment:
Can you leave a comment here that we leave it undefined so it doesn't think we're trying to set a custom axis title on the backend?